[2022-06-05 20:23:10,248] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:22:30.900766+00:00 [queued]>
[2022-06-05 20:23:10,265] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-05T23:22:30.900766+00:00 [queued]>
[2022-06-05 20:23:10,266] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:23:10,266] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 20:23:10,266] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 20:23:10,288] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-05 23:22:30.900766+00:00
[2022-06-05 20:23:10,298] {standard_task_runner.py:52} INFO - Started process 161583 to run task
[2022-06-05 20:23:10,303] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-05T23:22:30.900766+00:00', '--job-id', '985', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsa_x__ly', '--error-file', '/tmp/tmpit4_tivb']
[2022-06-05 20:23:10,304] {standard_task_runner.py:80} INFO - Job 985: Subtask populate_table
[2022-06-05 20:23:10,383] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-05T23:22:30.900766+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 20:23:10,494] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T23:22:30.900766+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T23:22:30.900766+00:00
[2022-06-05 20:23:11,229] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 20:23:11,252] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220605T232230, start_date=20220605T232310, end_date=20220605T232311
[2022-06-05 20:23:11,318] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 20:23:11,373] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
